packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -1674,7 +1691,7 @@ abstract class DiagnosticsNode {
       truncated = true;
     }
     final List<Map<String, Object?>> json = nodes.map<Map<String, Object?>>((DiagnosticsNode node) {
-      return node.toJsonMap(delegate.delegateForNode(node));
+      return node.toJsonMap(delegate.delegateForNode(node), fullDetails: fullDetails,);
```

Comment:
remove trailing comma if this will all be on one line, or format it with the trailing comma

Reply:
oops, fixed